defi/src/updateSearch.ts

```diff
@@ -249,10 +268,111 @@ const getProtocolSubSections = ({
   }));
 };
 
+async function getAllCurrentSearchResults() {
+  const res: { total: number; results: Array<SearchResult> } = await fetch(
+    "https://search.defillama.com/indexes/pages/documents",
+    {
+      headers: {
+        Authorization: `Bearer ${process.env.SEARCH_MASTER_KEY}`,
+      },
+    }
+  ).then((res) => res.json());
+  const finalRes: { total: number; results: Array<SearchResult> } = await fetch(
```

Comment:
i dont think this will work, what you need to do is fetch the new page in a loop till the number of items returned is smaller than pagesize

this is basic pagination